contact.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/dist/output.css" />
+    <title>Baumeister</title>
+  </head>
+  <script>
+    const loadComponents = (id, url) => {
+      fetch(url)
```

Comment:
Consider adding  error handling in the fetch function to log any errors encountered as its always a good practice.
It could be something like this:
 ```
 catch((error) => {
            console.error('There was a problem with the fetch operation:', error);
          })